pkg/cmd/release/create/create.go

```diff
@@ -1042,20 +1109,25 @@ outer_loop:
 		if err != nil {
 			return nil, nil, err
 		}
-		if answer == "u" { // undo!
+
+		if answer == "y" { // YES these are the packages they want
+			break
+		} else if answer == "?" { // help text
+			_, _ = fmt.Fprintf(stdout, prettifyHelp(packageOverrideLoopHelpText))
+		} else if answer == "u" { // undo!
 			if len(packageVersionOverrides) > 0 {
 				packageVersionOverrides = packageVersionOverrides[:len(packageVersionOverrides)-1]
 				// always reset to the baseline and apply everything in order, there's less room for logic errors
 				overriddenPackageVersions = ApplyPackageOverrides(packageVersionBaseline, packageVersionOverrides)
 			}
 			continue // print table and go again
-		}
-
-		if answer == "y" { // YES these are the packages they want
-			break
-		}
-
-		if resolvedOverride != nil {
+		} else if answer == "r" { // reset! All the way back to the calculated versions, discarding even the stuff that came in from the cmdline
+			if len(packageVersionOverrides) > 0 {
+				packageVersionOverrides = make([]*PackageVersionOverride, 0)
+				overriddenPackageVersions = ApplyPackageOverrides(packageVersionBaseline, packageVersionOverrides)
+			}
+			continue // print table and go again
+		} else if resolvedOverride != nil {
```

Comment:
This is starting to get bit big, maybe time to use a switch instead. They don’t have problems like falling through in go.

Reply:
I did that internally, but that it changes the semantics of `break` within the 'yes' section... so I'd need an outer break with a label... and then it's not clear that's an improvement anymore

Reply:
actually turns out we already had an outer break label, so the penalty doesn't apply; I've changed to a switch now and it's better, thanks